modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -10,7 +9,7 @@
 
 A _taint_ allows a node to refuse pod to be scheduled unless that pod has a matching _toleration_.
 
-You apply taints to a node through the node specification (`NodeSpec`) and apply tolerations to a pod through the `Pod` specification (`PodSpec`). A taint on a node instructs the node to repel all pods that do not tolerate the taint.
+You apply taints to a node through the `Node` specification (`NodeSpec`) and apply tolerations to a pod through the `Pod` specification (`PodSpec`). A taint on a node instructs the node to repel all pods that do not tolerate the taint.
```

Comment:
GH ate my comment, apparently.

I'm not familiar with a `NodeSpec` or a `PodSpec`. I've never seen these referred to in this way.

Reply:
Those are Kubernetes terms that appear sporadically in the docs. Here they are holdovers from the 3.11 docs. I can remove them here, if you think. 
https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#podspec-v1-core
https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#node-v1-core